<launch>
<!-- TODO: switch controllers -->
<group ns="mur620">
<!-- Launch the twist_sim node -->
    <node name="twist_sim" pkg="ur_controllers_match" type="twist_controller_fb" output="screen">
        <!-- Custom parameters -->
        <param name="prefix_ur" value="UR10_r/" />
        <param name="group_name" value="UR_arm_r" />
        
        <!-- Topic remapping -->
        <remap from="UR10_r/joint_group_vel_controller/command" to="/mur620/joint_group_vel_controller_r/safe/command" />
        <remap from="twist_command" to="UR10_r/twist_fb_command" />

    </node>
</group>
</launch>

